src/lang-promql/complete/hybrid.ts

```diff
@@ -59,25 +59,26 @@ const autocompleteNode: { [key: string]: AutoCompleteNode } = {
 
 const snippets: readonly SnippetSpec[] = [
   {
-    keyword: 'sum(rate(__input_vector__[5m]))',
+    label: 'sum(rate(__input_vector__[5m]))',
+    type: 'function',
     snippet: 'sum(rate(${__input_vector__}[5m]))',
   },
   {
-    keyword: 'histogram_quantile(__quantile__, sum by(le) (rate(__histogram_metric__[5m])))',
+    label: 'histogram_quantile(__quantile__, sum by(le) (rate(__histogram_metric__[5m])))',
+    type: 'function',
     snippet: 'histogram_quantile(${__quantile__}, sum by(le) (rate(${__histogram_metric__}[5m])))',
   },
   {
-    keyword: 'label_replace(__input_vector__, "__dst__", "__replacement__", "__src__", "__regex__")',
+    label: 'label_replace(__input_vector__, "__dst__", "__replacement__", "__src__", "__regex__")',
+    type: 'function',
     snippet: 'label_replace(${__input_vector__}, "${__dst__}", "${__replacement__}", "${__src__}", "${__regex__}")',
   },
 ];
 
 // parsedSnippets shouldn't be modified. It's only there to not parse everytime the above list of snippet
 const parsedSnippets: Completion[] = snippets.map((s) => ({
-  label: s.name || s.keyword,
-  original: s.name || s.keyword,
+  label: s.label,
   apply: snippet(s.snippet),
-  score: 0,
```

Comment:
Please don't think this is a review, I just have a question. After this PR we still don't have scoring in this hybrid mode, right? I mean we haven't used CM's [boost](https://codemirror.net/6/docs/ref/#autocomplete.Completion.boost) option yet?

Reply:
actually the scoring is fully integrated in the autocomplete module. The `boost` is just a field to say "this particular label is really important". It's a way to influence a bit the internal scoring system.

For the moment I don't see with what condition we should use it, but you can be sure the list returned is using an internal score.

Reply:
oh thank you. I've just went and check the autocomplete module and I saw the internal scoring system :+1: